.gitignore

```diff
@@ -83,11 +83,15 @@ bench-report.xml
 ##############
 
 /enso
+/project-manager
```

Comment:
Why do we need `project-manager` directory in addition to the `built-project-manager`?

Reply:
`project-manager` is not a directory, it's the name of the executable when building the project manager on UNIX platforms.

Reply:
And `built-project-manager` actually represents a built distribution package (maybe I should rename it to `distribution-packages/project-manager`; while the `project-manager` executable is just an executable that we build, similarly as `runner.jar` etc. that we keep in the root.